src/System.Drawing.Common/src/System/Drawing/GdiplusNative.Unix.cs

```diff
@@ -0,0 +1,2508 @@
+﻿using System.Drawing.Drawing2D;
+using System.Drawing.Imaging;
+using System.Drawing.Text;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.ComTypes;
+using System.Text;
+
+namespace System.Drawing
+{
+    internal partial class SafeNativeMethods
+    {
+        internal unsafe partial class Gdip
+        {
+            private const string LibName = "libdl";
+
+            public const int RTLD_NOW = 0x002;
+
+            [DllImport(LibName)]
+            public static extern IntPtr dlopen(string fileName, int flags);
+
+            [DllImport(LibName)]
+            public static extern IntPtr dlsym(IntPtr handle, string name);
```

Comment:
Nit: we should use the same names as the platform does, e.g. name => symbol, fileName => filename, etc.